### PR TITLE
#161340167 Integrate travis with a readme badge

### PR DESCRIPTION
**Pull request by nakatuddesuzan**
#### What does this PR do?
Integrate Travis  with a readme badge
#### Description of Task to be completed?
Added Travis.yml file
Added Travis badge to the Readme file
#### What are the relevant pivotal tracker stories?
[#161340167](https://www.pivotaltracker.com/story/show/161340167)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: python
+python:
+- "3.7"
```
Comment (daud1) [inline .travis.yml]: Python3.7 is not yet supposed by most build tools because it is pretty recent.
Please change this to a stable version of Python.

Comment (daud1) [inline .travis.yml]: Python3.7 is not yet supposed by most build tools because it is pretty recent.
Please change this to a stable version of Python.

**Review by daud1 — CHANGES_REQUESTED** [review]
Python3.7 is not yet supposed by most build tools because it is pretty recent.
Please change this to a stable version of Python.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/andela/ah-backend-odin.svg?branch=develop)](https://travis-ci.org/andela/ah-backend-odin)
```
Comment (krmroland) [inline README.md:1]: The travis badge is currently pointing to the develop branch, which i think is okay.  My question is if the master(stable branch) should be displaying the develop build's badge.

**Review by krmroland — COMMENTED** [review]
Insight

**Review by daud1 — APPROVED** [review]

**Pull request merged**